src/scripts/main.js

```diff
@@ -6,3 +6,32 @@ const people = require('./lib/people');
 console.log(people); // you can remove it
 
 // write your code here
+const table = document.querySelector('.dashboard');
+
+const peopleForTable = people.map(person => {
+  const century = Math.ceil(person.died / 100);
+  const age = person.died - person.born;
+  const sex = person.sex === 'm' ? 'Male' : 'Female';
+
+  return ({
+    name: person.name,
+    sex: sex,
+    born: person.born,
+    died: person.died,
+    age: age,
+    century: century,
+  });
+});
+
+peopleForTable.forEach(person => {
+  table.insertAdjacentHTML('beforeend', `
```

Comment:
your table looks a bit strange: 
![image](https://user-images.githubusercontent.com/86923089/208072338-7336ee17-b6bd-41b0-80fd-673f1d22b4f1.png)

Probably because of way you added rows
Fix it, plesae